docs/content/engines/quick_start.md

```diff
@@ -26,43 +26,138 @@ under the License.
 
 
 # Qucik Start
-This document explains how to use Docker to quickly build a 
-practice environment for Paimon and big data engines (Flink,
-Spark, Hive, etc.).
+This document describes how to use Docker and Docker Compose 
+to quickly build a big data environment such as Paimon and 
+the big data engine Flink/Spark. 
+You can see more runtime information between Paimon and 
+Flink/Spark by looking at the corresponding section in the 
+documentation.
 
 The following steps have been tested on a Windows 11 laptop 
-with docker desktop installed. 
+with docker desktop and docker compose installed.
 
 ## Prerequisites
-windows 11
+windows
 
 docker desktop for windows
 
-## Use paimon in flink
-flink version: 1.17.x
+docker compose
+
+## Quick start paimon and flink
+
+Look at the version mapping between paimon and Flink 
+in the [Engines/Flink](https://paimon.apache.org/docs/master/engines/flink/) section of the document and select 
+the version you want
+
+### Docker compose
+
+docker compose is a convenient way to manage docker containers and quickly build the application environment 
+you need. The following docker compose file is composed of three services, jobmanager, taskmanager and sql-client
+, using the flink:1.17.1-scala_2.12 image.
+
+Copy the following into a ```flink_docker_compose.yml``` file
```

Comment:
Nit:
"Create a file named flink_docker_compose.yml and copy the following into it" might be a better way to deliver the same meaning

Reply:
Your suggestion is very good, it does make the description more accurate

Reply:
new commit
https://github.com/apache/incubator-paimon/pull/1832/commits/a131bc5f0a870d09601f75d1d90deab9393e99a8